docs/CodeDevelopment/requirements.md

```diff
@@ -32,21 +32,50 @@
         }
     ```
 
-## Uefi Package
+## Repositories
```

Comment:
Repositories and packages should have an altitude or layer or position.  
That should be documented as that provides a framework for what is an acceptable dependency.  I have never found a word that describes this well.  Any ideas?  An example.  Per our philosophy it doesn't make sense to have Q35Pkg with MdePkg in the same repo.   How do we describe that?

Reply:
docs/faq.md

Added section attempting to address altitude and layering. 

It's more difficult than I thought to try to condense into just a few paragraphs